bin/mozilla-en-US.py

```diff
@@ -115,10 +115,18 @@ def push(path):
 dname = os.path.dirname(abspath)
 os.chdir(dname)
 
-for channel in ['aurora', 'beta']:
+for channel in ['beta', 'central']:
     for repo in SOURCE_REPOS:
         ending = repo + '-' + channel
-        url = 'ssh://hg.mozilla.org/releases/' + ending
+
+        url_folder = ''
+        if channel == 'central':
```

Comment:
I agree. This script will go away as soon as we land x-channel, so I'd rather avoid touching it at this stage.